### PR TITLE
Move man doc links to descriptions, fix some help copies

### LINKED ISSUE #1798: dvc: help: move doc links to long description
```
Available Commands:
  COMMAND        Use dvc COMMAND --help for command-specific help.
    init         Initialize dvc over a directory (should already be a git dir).
                 documentation: https://man.dvc.org/init
    destroy      Destroy dvc. Will remove all repo's information, data files and cache.
                 documentation: https://man.dvc.org/destroy
    add          Add files/directories to dvc.
                 documentation: https://man.dvc.org/add
    remove       Remove outputs of DVC file.
                 documentation: https://man.dvc.org/remove
    move         Rename or move a DVC controlled data file or a directory.
                 documentation: https://man.dvc.org/move
    unprotect    Unprotect data file/directory.
                 documentation: https://man.dvc.org/unprotect
    run          Generate a stage file from a given command and execute the command.
                 documentation: https://man.dvc.org/run
    repro        Reproduce DVC file. Default file name - 'Dvcfile'.
                 documentation: https://man.dvc.org/repro
    pull         Pull data files from the cloud.
                 documentation: https://man.dvc.org/pull
    push         Push data files to the cloud.
                 documentation: https://man.dvc.org/push
    fetch        Fetch data files from the cloud.
                 documentation: https://man.dvc.org/fetch
    status       Show the repo status.
                 documentation: https://man.dvc.org/status
    gc           Collect garbage from your DVC cache.
                 Deletes all files in the cache which are not in use by the
                 specified git references (defaults to just HEAD).
                 documentation: https://man.dvc.org/gc
    add          Add files/directories to dvc.
                 documentation: https://man.dvc.org/add
    import       Import files from URL.
                 documentation: https://man.dvc.org/import
    config       Get or set config options.
                 documentation: https://man.dvc.org/config
    checkout     Checkout data files from cache.
                 documentation: https://man.dvc.org/checkout
    remote       Manage set of tracked repositories.
                 documentation: https://man.dvc.org/remote
    cache        Manage cache settings.
                 documentation: https://man.dvc.org/cache
    metrics      A set of commands to add, manage, collect and display project metrics.
                 documentation: https://man.dvc.org/metrics
    install      Install dvc hooks into the repository.
                 documentation: https://man.dvc.org/install
    root         Relative path to repo's directory.
                 documentation: https://man.dvc.org/root
    lock         Lock DVC file.
                 documentation: https://man.dvc.org/lock
    unlock       Unlock DVC file.
                 documentation: https://man.dvc.org/unlock
    pipeline     Manage pipeline.
                 documentation: https://man.dvc.org/pipeline
    commit       Record changes to the repository.
                 documentation: https://man.dvc.org/commit
```

### PR DESCRIPTION
**Pull request by shcheklein**
Fixes #1798

### CODE REVIEW

File: dvc/command/init.py [hunk 1]
```diff
@@ -44,6 +52,6 @@ def add_parser(subparsers, parent_parser):
         "--force",
         action="store_true",
         default=False,
-        help="Overwrite '.dvc' if it exists. Will remove all local cache.",
+        help="Overwrite '.dvc' if it exists. It removes local cache.",
```
Comment (efiop) [inline dvc/command/init.py]: The previous message feel more clear, tbh 🙂 Using "It" here feels unnatural.

File: dvc/command/lock.py [hunk 2]
```diff
@@ -22,29 +24,34 @@ def run(self, unlock=False):
 
 
 class CmdLock(CmdLockBase):
-    def run(self):
+    def run(self, _=False):
```
Comment (efiop) [inline dvc/command/lock.py]: Not sure what this is for.

Comment (efiop) [reply dvc/command/lock.py]: Same in run() below.

Comment (shcheklein) [reply dvc/command/lock.py]: something like this - https://help.semmle.com/wiki/display/PYTHON/Signature+mismatch+in+overriding+method

Comment (efiop) [reply dvc/command/lock.py]: @shcheklein Not a big fan of such things. Could we at least put a comment about it? Right now it looks like a left-over. Or maybe even rename CmdLockBase.run() into CmdLockBase._run() so we don't even have to bother with this stuff.

Comment (shcheklein) [reply dvc/command/lock.py]: Not big fan either, don't like potentially breaking the interface though. Let me take a look if there are other options and what ppl do about it. What do you mean by renaming? I'm not sure I understand you suggestion with renaming.

Comment (efiop) [reply dvc/command/lock.py]: @shcheklein So CmdLock inherits from CmdLockBase, which has a method called run() with two args. This is where that pycharm warning comes from. If we just rename it to _run() and use self._run() in CmdLock/Unlock, then that warning should disappear.

File: dvc/command/lock.py [hunk 3]
```diff
@@ -23,28 +25,33 @@ def run(self, unlock=False):
 
 class CmdLock(CmdLockBase):
     def run(self):
-        return super(CmdLock, self).run(False)
+        self._run(False)
```
Comment (efiop) [inline dvc/command/lock.py]:
```suggestion
        return self._run(False)
```

File: dvc/command/lock.py [hunk 4]
```diff
@@ -23,28 +25,33 @@ def run(self, unlock=False):
...
 
 
 class CmdUnlock(CmdLockBase):
     def run(self):
-        return super(CmdUnlock, self).run(True)
+        self._run(True)
```
Comment (efiop) [inline dvc/command/lock.py]:
```suggestion
        return self._run(True)
```

**Pull request merged**